src/main/resources/reportFile.csv

```diff
@@ -0,0 +1,3 @@
+fruit,quantity
+banana,152

```

Comment:
132

Reply:
I don't know how you got this number. In README this was the example.
![image](https://user-images.githubusercontent.com/118270678/212695694-8c13622f-91ef-42cb-b3a9-aff52a4f2938.png)

Reply:
my mistake - missed apple with banana)